BP/common/religion/_ToDo.info

```diff
@@ -0,0 +1,60 @@
+To Do Tasks:
+Fix some remaining errors:
+	[12:18:33][jomini_dynamicdescription.cpp:66]: Unrecognized loc key special_doctrine_christian_ecumenism_name. special_doctrine_christian_ecumenism
```

Comment:
Do you plan on doing this before or after merge?

Reply:
Fixed already; forgot to remove this in the to-dos file.